Assets/Scripts/Interaction/PlayerInteractor.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using Interaction.Core;
+using Inventory.Custom;
+using Mirror;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+namespace Interaction
+{
+    [RequireComponent(typeof(Hands))]
+    public class PlayerInteractor : NetworkBehaviour
+    {
+        private new Camera camera;
+        private Hands hands;
+        
+        private void Start()
+        {
+            if (!isLocalPlayer) Destroy(this);
+            hands = GetComponent<Hands>();
+        }
+
+        protected virtual void Update()
+        {
+            if (!camera) camera = Camera.main;
+            if (!camera) return;
+            
+            if (Input.GetMouseButtonDown(0))
```

Comment:
Remove hardcoded button, use input mapping.

Reply:
This is the same code as we had before. We need to agree on how input should be managed first

Reply:
This is the same code as we had before. We need to agree on how input should be managed first